packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -761,6 +761,48 @@ class EditableText extends StatefulWidget {
   /// To be notified of all changes to the TextField's text, cursor,
   /// and selection, one can add a listener to its [controller] with
   /// [TextEditingController.addListener].
+  ///
+  /// {@tool dartpad --template=stateless_widget_material}
+  ///
+  /// This example shows how onChanged could be used to check the value in the
```

Comment:
... to check the textfield's current value each time the user inserts or deletes a character.